cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -534,32 +588,23 @@ function(deal_ii_add_test _category _test_name _comparison_file)
         TIMEOUT ${TEST_TIME_LIMIT}
         )
 
+      if(_shared_target)
+        set_tests_properties(${_test_full} PROPERTIES
+          FIXTURES_REQUIRED ${_test_executable_full}
+          )
+      endif()
+
       if(_exclusive)
         #
         # Ensure that the test is not executed concurrently with any other
         # tests.
         #
         set_tests_properties(${_test_full} PROPERTIES RUN_SERIAL TRUE)
 
-      elseif(NOT ENABLE_PERFORMANCE_TESTS)
+      elseif()
```

Comment:
Wait? Did you accidentally read this as an `else(CONDITION)`?

Reply:
This is why I love CMake so much. Fixed.

Reply:
Ha, are you saying I pointed out an actual bug?